infrastructure/deployment.json

```diff
@@ -50,6 +50,12 @@
         "description": "The pricing tier for the App Service plan"
       }
     },
+    "principalId": {
```

Comment:
Is this the System Assigned Identity? Should this be the output of the web app identity?